### PR TITLE
feat: add plugin meta data

### PR DESCRIPTION
**Pull request by kalilistic**
List based on some discord discussions.

### CODE REVIEW

**Review by goaaats — APPROVED** [review]

File: UIRes/pluginmetadata.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    "Name": "ACTEndcounter",
+    "DevSupportState": "Adoptable"
+  },
+  {
+    "Name": "BrowserHost.Plugin",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Browsingway"
+  },
+  {
+    "Name": "CharSyncPlugin",
```
Comment (reiichi001) [inline UIRes/pluginmetadata.json:17]: This was actually a replacement/alternative for Character Data Sync iirc. It's outdated, but I'm not sure this reason matches up officially.

File: UIRes/pluginmetadata.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Character Data Sync"
+  },
+  {
+    "Name": "ChatExtender",
```
Comment (reiichi001) [inline UIRes/pluginmetadata.json:22]: We should probably note that ChatExtender was actually broken up into separate plugins.
- Chat bubbles are handled by Chat Bubbles (Haplo)
- Chat translation is handled by Chat Translator (Haplo)
- Chat logging was replaced by XIVLogger (Cadaeix)

File: UIRes/pluginmetadata.json [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    "Name": "TabTab",
+    "DevSupportState": "Adoptable"
+  },
+  {
+    "Name": "VoidList",
+    "DevSupportState": "Obsolete",
+    "DevSupportStateReason": "Replaced by Visibility"
+  },
+  {
+    "Name": "WheresWOLdo",
```
Comment (reiichi001) [inline UIRes/pluginmetadata.json:73]: Franz is still maintaining this (for now). It should be marked as maintenance-only instead

**Comment by kalilistic** [comment]
Updated based on Franz's feedback.

**Comment by kalilistic** [comment]
Closing as stale. I'll leave the branch on my fork if there's interest in picking up.

**Pull request closed without merging**